packages/e2e-tests/test-applications/generic-ts3.8/package.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "@sentry-internal/ts3.8-test",
+  "private": true,
+  "license": "MIT",
+  "scripts": {
+    "build:types": "pnpm run type-check",
+    "ts-version": "tsc --version",
+    "type-check": "tsc --project tsconfig.json",
+    "test:build": "pnpm install",
+    "test:assert": "pnpm run build:types"
```

```suggestion
    "test:build": "pnpm install && pnpm run build:types",
    "test:assert": "pnpm -v"
```

let's build everything in the build command, then noop assert (like in other test apps)

---

Sound good. Thanks.